jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/internal/ContentCopier.java

```
@@ -61,14 +61,7 @@ protected Action process() throws Throwable
             return Action.SCHEDULED;
 
         if (Content.Chunk.isFailure(current))
-        {
-            if (current.isLast())
-                throw current.getFailure();
-            if (LOG.isDebugEnabled())
-                LOG.debug("ignored transient failure", current.getFailure());
-            succeeded();
-            return Action.SCHEDULED;
-        }
+            throw current.getFailure();
```

If the read a transient failure, I think we should behave as if we read `null` for the goal of copying.

---

I disagree as that would silently ignore idle timeouts.